tools/publisher/src/subcommand/fix_manifests.rs

```diff
@@ -125,15 +126,36 @@ fn fix_dep_sets(versions: &BTreeMap<String, Version>, metadata: &mut toml::Value
     Ok(changed)
 }
 
+fn fix_publish_flag(manifest_path: &Path, metadata: &mut toml::Value) -> Result<bool> {
```

Comment:
nit: naming: I'd name this something a bit more specific `fix` implies it's broken, maybe `block_local_publish` or something?